examples/src/bin/limitedclient.rs

```diff
@@ -53,6 +53,15 @@ fn main() {
         ciphersuite.suite()
     )
     .unwrap();
+    let kx_group = tls
```

Comment:
On balance, not convinced of the utility of adding this to every example? It's taken this long for someone to ask about the functionality which indicates to me that it's a much more rare requirement (compared to cipher suite).

Reply:
I think I'd like to keep it in the `rustls-post-quantum` client example where the KX choice is the fundamental point of the example. I'll remove the other instances :+1:

Reply:
Removed from the other examples.